src/ResourceManager/Billing/Commands.Billing/Microsoft.Azure.Commands.Billing.dll-Help.psd1

```diff
@@ -0,0 +1,109 @@
+#
```

Comment:
I created this file because I saw other RPs have it. I have followed the platyPS doc to create help files. I think I should just remove the file.